main.go

```diff
@@ -28,43 +26,6 @@ func init() {
 	viper.AutomaticEnv()
 }
 
-func help(cmd *cobra.Command, args []string) {
```

Comment:
Is the removal of the help page in `main.go` intentional? I am not familiar with Go will it automatically be generated by Cobra?

Reply:
Yeah, it's unnecessary. `apx --help` works just as well without this as Cobra generates it automatically from `cmd/root.go`.